### PR TITLE
Add Iowa City Nemesis createPropertyType code

### PR DESCRIPTION
**Pull request by jmvx**

### CODE REVIEW

**Comment by jmvx** [comment]
Yes, this code was only used for creating a massive number of new property types. Is there another place I could move these files?

File: iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java [hunk 1]
```diff
@@ -0,0 +1,218 @@
+package com.dataloom.integrations.iowacity;
+
+import org.apache.olingo.commons.api.edm.FullQualifiedName;
+
+public class Nemesis {
```
Comment (UnsungHero97) [inline iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java]: It's good practice to set a private constructor in classes like these where everything is just a constant:

```
public class Nemesis {
  private Nemesis() {}
}
```

File: iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java [hunk 2]
```diff
@@ -0,0 +1,218 @@
...
+    static FullQualifiedName DEM_01  = new FullQualifiedName( "iowanational", "dagency01" );
```
Comment (UnsungHero97) [inline iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java]: It's good practice to set `public` and `final` on constants like these:

```
public static final FullQualifiedName DEM_01 = new FullQualifiedName( "iowanational", "dagency01" );
```

File: iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java (hunk 1 above)
Comment (jmvx) [inline iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java]: Added!

File: iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java (hunk 2 above)
Comment (jmvx) [inline iowacity/src/main/java/com/dataloom/integrations/iowacity/Nemesis.java]: Thank you! Changed them to be public static final constants.

File: iowacity/src/main/java/com/dataloom/integrations/iowacity/CreateNemesisProperties.java [hunk 3]
```diff
@@ -0,0 +1,1818 @@
...
+import com.dataloom.client.RetrofitFactory.Environment;
+
+import static com.dataloom.integrations.iowacity.Nemesis.*;
+
+public class CreateNemesisProperties {
+
+    public static void main( String[] args ) throws InterruptedException, ExecutionException {
+
+        String jwtToken = "replace with jwtToken";
+        LoomClient loomclient = new LoomClient( Environment.PRODUCTION, () -> jwtToken );
```
Comment (UnsungHero97) [inline iowacity/src/main/java/com/dataloom/integrations/iowacity/CreateNemesisProperties.java]: Let's be safe and default to `Environment.LOCAL`.

Comment (jmvx) [inline iowacity/src/main/java/com/dataloom/integrations/iowacity/CreateNemesisProperties.java]: Changed

**Pull request merged**